static.md

```diff
@@ -20,3 +20,22 @@ module.exports = {
     }]
 };
 ```
+
+If you only want to copy _some_ of the files,
+you can select them using a `filter` predicate.
+The predicate will be called for every file with its path relative to `source`.
+Each file it returns `true` for gets copied.
```

Comment:
This seems a little hard to parse; perhaps this scans better:

> Only files for which that function returns `true` are copied.

Alternatively:

> Any return value other than `true` will result in the respective file being omitted / _not_ being copied.